mk-files/docs.mk

```
@@ -67,12 +67,14 @@ release-docs-main:
 	$(call dry-run,git push -u origin $(CURRENT_SHORT_MINOR_VERSION)-post) && \
 	if [[ $(CLEAN_VERSION) == *.0 ]]; then \
 		git checkout master && \
+		git checkout -b release-docs-main-v$(CLEAN_VERSION) && \
```

Looking at our other PRs, we leave the leading "v" out
```suggestion
		git checkout -b release-docs-main-$(CLEAN_VERSION) && \
```